src/main/java/RacingGame.java

```diff
@@ -0,0 +1,70 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+//자동차 목록, 한 라운드의 자동차 위치 등
+public class RacingGame {
+    private final Scanner scanner = new Scanner(System.in);
+    private final List<Car> cars = new ArrayList<>();
+
+    public String[] nameSplit(String str) {
+        String[] nameArray;
+        nameArray = str.split(",");
+        return nameArray;
+    }
+    public boolean nameCheck(String[] names) {
+        for(int i = 0; i < names.length; i++) {
+            if(names[i].length() > 5 || names[i].length() <= 0) {
+                System.out.println((i + 1) + "번째 이름을 5자리 이하로 입력하세요");
+                return true;
+            }
+        }
+        return false;
+    }
+    public int maxPositionIndex() {
+        int maxIndex = 0;
+        for(int i = 0; i < cars.size(); i++) {
+            if(cars.get(maxIndex).getPosition() <= cars.get(i).getPosition()) {
+                maxIndex = i;
+            }
+        }
+        return maxIndex;
+    }
+    public String winnerPrint(int max) {
+        String winCar = "";
+        for(int i = 0; i < max; i++) {
+            if(cars.get(max).getPosition() == cars.get(i).getPosition()) {
+                winCar += cars.get(i).getName() + ", ";
+            }
+        }
+        winCar += cars.get(max).getName() + "가 최종 우승했습니다";
+        return winCar;
+    }
+    public void addCarList(String[] name) {
```

Comment:
```suggestion
    public void addCar(String[] name) {
```
List, Map 같은 불용어는 안쓰는게 좋아요~